src/material/menu/menu.md

```diff
@@ -4,16 +4,9 @@
 
 By itself, the `<mat-menu>` element does not render anything. The menu is attached to and opened
 via application of the `matMenuTriggerFor` directive:
-```html
-<mat-menu #appMenu="matMenu">
-  <button mat-menu-item>Settings</button>
-  <button mat-menu-item>Help</button>
-</mat-menu>
-
-<button mat-icon-button [matMenuTriggerFor]="appMenu">
-  <mat-icon>more_vert</mat-icon>
-</button>
-```
+<!-- example({"example":"menu-overview",
+              "file":"menu-overview-example.html",
```

Comment:
Everywhere else we have a space after the colon in object literals, we should probably do the same here too.